tools/omero/omero_get_value.py

```diff
@@ -1,31 +1,33 @@
 import argparse
 import csv
 import json
+import os
 import sys
 
 import ezomero as ez
+import pandas as pd
 
 
-def get_object_ezo(user, pws, host, port, obj_type, ids, tsv_file):
+def get_object_ezo(user, pws, host, port, obj_type, ids, out_dir):
     # Function to write tabular file from the ezomero output
     def write_values_to_tsv(data, header):
-        with open(tsv_file, 'w', newline='') as f:
+        with open("output.tsv", 'w', newline='') as f:
             writer = csv.writer(f, delimiter='\t')
             writer.writerow([header])  # Write the header
             for item in data:
                 writer.writerow([item])  # Write each value
 
     # Function to write tabular file from a dictionary ezomero output
     def write_dict_to_tsv(data, headers):
-        with open(tsv_file, 'w', newline='') as f:
+        with open("output.tsv", 'w', newline='') as f:
             writer = csv.writer(f, delimiter='\t')
             writer.writerow(headers)  # Write the headers
             for key, value in data.items():
                 writer.writerow([key, value])  # Write each key-value pair
 
     # Function to write tabular file from list of list ezomero output
-    def write_table_to_tsv(data):
-        with open(tsv_file, 'w') as f:
+    def write_table_to_tsv(data, file_name):
+        with open(f"./output/ID_{file_name}_table.tsv", 'w') as f:
```

Comment:
The `_table` will show up in the element identifiers of the collection, is this desired? 

```suggestion
       with open(f"./output/ID_{file_name}.tsv", 'w') as f:
```

Are in this case the `file_name`s unique? Otherwise you may need to add the ID as well -- like in the attachment case?

Reply:
ah, there is a bit of confusion here: `file_name` is the file ID, i will change the variable to avoid confusion